src/Controller.php

```diff
@@ -46,6 +47,12 @@ public function connect(Application $app)
             ->assert('file', '.+')
             ->bind('thumb');
 
+        // Aliases
+        $ctr->get('/{alias}/{file}', 'controller.thumbnails:alias')
+            ->assert('alias', '[a-zA-Z0-9-_]+')
+            ->assert('file', '.+')
+            ->bind('alias');
```

Comment:
routing bindings are global, I think it needs to be more specific than `alias`. Maybe `thumb_alias` or `thumbnail_alias`?